packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -389,4 +389,14 @@ module.exports = {
       }
     }
   },
-}
+  async ready() {
+    if (!ctx.isProd) {
```

Comment:
As mentioned in Slack, this won't work since both the Trex and Prod environment deploys use the "production" webpack build, which will result in `ctx.isProd = true`